content/en/about/codeofconduct.md

```diff
@@ -28,44 +28,30 @@ When deciding on partners to cooperate with, our ethics commission will place gr
 # Our working principles
 
 
-### Flexibility
-
-We guarantee that both NPOs and volunteers are able to freely dedicate their time to projects to accommodate for different needs. We believe that this enables us to get everyone involved and make the best out of the experience.
-
+### Self-determination
+Volunteers and partners work and make decisions self-determined and responsibly. To accommodate for different working styles and backgrounds, we do not prefer specific coding languages or technologies but rather encourage our partners and members to choose the right path for themselves, guiding them through their unique journey.
 
 ### Pragmatism
 
 We create practical and sustainable solutions that have real value to our partners. We do so by striving to use technologies that can be implemented, used, maintained and updated by our partners, even if that means that we have to integrate less advanced tools. We foster an impact-driven mindset for the greater good.
 
-
-### Professionalism
-
-We dedicate ourselves to have high working standards and ensuring that insights derived from our analysis are profound. Data protection and security are our key concerns. We especially make sure that confidential data is treated carefully and deleted after project closure.
-
-
 ### Transparency
 
 We are transparent in how we derive conclusions and try to explain methods and technologies used to our partners to empower them on their individual data journey. We also present differing views and are honest about the limitations of our knowledge and work. 
 
+We believe in sharing our work, knowledge and experience. We make code and resources publicly available whenever possible and attend conferences and talks to share our expertise. 
 
-### Appreciation
-
-We respect and value other people and their views, especially if they are different from ours. We do so by learning from each other and practicing gratitude. This can be as simple as saying ‘thank you’.
-
-
-### Diversity 
-
-We embrace diversity in all its forms and take care of fostering tolerance and acceptance inside our community and beyond.
-
+Information and decision making processes within CorrelAid are accessible and open to as many people as possible. We work towards removing knowledge silos by sharing and documenting workflows, processes and knowledge. 
```

Comment:
We make code and resources publicly available whenever possible and attend conferences and talks to share our expertise.